com.xunlei.Thunder.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.xunlei.Thunder</id>
+  <launchable type="desktop-id">com.xunlei.Thunder.desktop</launchable>
+  <name>Thunder</name>
+  <developer_name>xunlei</developer_name>
+  <summary>Xunlei download. 迅雷下载。</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://i.xunlei.com/xluser/legal/agreement.html</project_license>
+  <url type="homepage">https://www.xunlei.com/</url>
+  <description>
+    <p>Xunlei download, which supports downloading HTTP, BitTorrent, Magnet, etc.</p>
```

Comment:
Could you include a note that this is not supported by upstream, similar to this one for Slack. Other than that, lgtm

https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L18

Reply:
Thank you for the suggestion, I have added it through [this commit](https://github.com/qvshuo/flathub/commit/a441ac3179ef33a5fb6c5df55532982a301c9d60).